### ISSUE TITLE
wallabag:install, clear:cache error

### ISSUE THREAD

**Issue by iGormilhit** [wallabag/wallabag#2265]
I'm trying to install wallabag on a debian wheezy dedicaded server. PHP is in version 5.6 and the database is MySQL.

I have followed the following instructions:
```
git clone https://github.com/wallabag/wallabag.git
cd wallabag
git checkout 2.0.7
SYMFONY_ENV=prod composer install --no-dev -o --prefer-dist
php bin/console wallabag:install --env=prod
```

When running `php bin/console wallabag:install --env=prod` I get the following:

```
Installing Wallabag...

Step 1 of 5. Checking system requirements.
+-----------------+--------+----------------+
| Checked         | Status | Recommendation |
+-----------------+--------+----------------+
| PDO Driver      | OK!    |                |
| curl_exec       | OK!    |                |
| curl_multi_init | OK!    |                |
+-----------------+--------+----------------+
Success! Your system can run Wallabag properly.

Step 2 of 5. Setting up database.
Creating database and schema
Clearing the cache
The command "cache:clear" terminated with an error code: 1.

                                                               
  [Exception]                                                  
  The command "cache:clear" terminated with an error code: 1.  
                                                               

cache:clear [--no-warmup] [--no-optional-warmers] [-h|--help] [-q|--quiet] [-v|vv|vvv|--verbose] [-V|--version] [--ansi] [--no-ansi] [-n|--no-interaction] [-e|--env ENV] [--no-debug] [--] <command>
```
I still haven't found a way to understand what is this error.

When I open the URL pointing to my wallabag instance, I have an blank page. I don't know it it's related to the cache or if I have other problems.

**Comment by nicosomb** [comment]
Hello @iGormilhit!

Can you try to clear cache manually (rm -rf var/cache/*) and relaunch wallabag installation?

**Comment by iGormilhit** [comment]
Hello @nicosomb and thanks for your answer.

I'd like too, but this "dedicated" server is running some other things than wallabag, so I find a bit brutal to clear the complete /var/cache/* ...

**Comment by iGormilhit** [comment]
Ok, so now I cleared the var/cache directory, restarted the installation. The new error message I get is related to database : The command "doctrine:database:drop" terminated with an error code: 1.

It may be because of the database_path of the parameters.yml. In case of a mysql database, I'm not quite sur what path to give there.

**Comment by tcitworld** [comment]
We were speaking of the `var/cache` directory inside wallabag, not the `/var/cache` of your server.

**Comment by j0k3r** [comment]
`database_path` should be set to `null` for MySQL.

Could you manually run `php bin/console doctrine:database:drop -e=prod` and told us what you get?

**Comment by iGormilhit** [comment]
@j0k3r OK, I've changed the `database_path` to `null` and launched `php bin/console doctrine:database:drop -f -e=prod`. Here's what I got:

```
php bin/console doctrine:database:drop -e=prod
ATTENTION: This operation should not be executed in a production environment.

Would drop the database named /var/www/poche/app/../data/db/wallabag.sqlite.
Please run the operation with --force to execute
All data will be lost!
```

I'm surprised, because in `app/config/parameters.yml`, I have:

```
database_driver: pdo_mysql
    database_host: localhost
    database_port: 3306
    database_name: xxxx
    database_user: xxxx
    database_password: xxxx
    database_path: null
    database_table_prefix: wallabag_
```

**Comment by j0k3r** [comment]
Yeah this is strange, might be a cache problem.
Run `php bin/console cache:clear -e=prod`

**Comment by iGormilhit** [comment]
Clearing the cache solved the problem concerning the database installation. Now the `php bin/console wallabag:install --env=prod` succeded, and I have been asked to set an admin account, which I did.

I have some warnings: `It is not safe to rely on the system's timezone settings. You are *required* to use the da  
  te.timezone setting or the date_default_timezone_set() function. In case you used any of those methods and you are   
  still getting this warning, you most likely misspelled the timezone identifier. We selected the timezone 'UTC' for   
  now, but please set date.timezone to select your timezone.`

But still, I get a blank page, even on the wallabag login. I've check the apache error log, and also I've check that all the php modules needed are installed and enabled.

**Comment by iGormilhit** [comment]
I think this is an apache configuration problem.

**Comment by j0k3r** [comment]
The message about the timezone is quite common: https://github.com/wallabag/wallabag/issues/1591#issuecomment-172771830

Did you try to run the `php bin/console server:run -e=prod` and then open http://127.0.0.1:8000 to see if wallabag is running.

**Comment by iGormilhit** [comment]
Now I did it, and no, wallabag isn't running.

**Comment by j0k3r** [comment]
What you got when you head to the url I gave you after launching the command?

**Comment by iGormilhit** [comment]
If I tried to reach the local server's IP on port 8000, then I get a "failed connexion". If I try, in the server console to reach http://127.0.0.1:8000 (with lynx), then I have an error 500.

**Comment by j0k3r** [comment]
Could you check what's in `var/log/*`?

**Comment by iGormilhit** [comment]
Nice, there are some logs ! :)

`cache/prod/annotations` wasn't writable, and now it is. I've relauched the php server, but I don't have more messages.

From the already written messages in `var/logs/prod.log`, I can notice this, but I don't think it's the problem:
`[2016-09-04 15:38:11] request.CRITICAL: Uncaught PHP Exception Twig_Error_Runtime: "An exception has been thrown during the rendering of a template ("Setting "piwik_enabled" couldn't be found.") in "WallabagCoreBundle::base.html.twig" at line 73." at /var/www/wallabag/var/cache/prod/classes.php line 1414 {"exception":"[object] (Twig_Error_Runtime(code: 0): An exception has been thrown during the rendering of a template (\"Setting \"piwik_enabled\" couldn't be found.\") in \"WallabagCoreBundle::base.html.twig\" at line 73. at /var/www/wallabag/var/cache/prod/classes.php:1414, RuntimeException(code: 0): Setting \"piwik_enabled\" couldn't be found. at /var/www/wallabag/vendor/craue/config-bundle/Util/Config.php:139)"} []`

**Comment by j0k3r** [comment]
Did you use a fresh database or an existing one?
Did you run the `php bin/console wallabag:install -e=prod` command?
It seems that some internal configuration aren't found in the database.

**Comment by iGormilhit** [comment]
I used a fresh database, which I created myself, and when I launched the install script (`php bin/console wallabag:install -e=prod`) I was asked it I wanted to overwrite it, to which I said yes. Should I drop this database and relaunch the install script ?

**Comment by j0k3r** [comment]
Might be a good idea.
Let the installer do the work for you.

**Comment by iGormilhit** [comment]
Then so I did. Drop the existing database, lauched the install script, the built-in server and got locally the login page. Then I tried to reach the server through my domain name, without success. I've checked the logs and saw that the cache/prod/annotations wasn't writabe. I've corrected it, and now I can log in.

Thank you very much and sorry for having taken you time... It's not the first time I'm struggling with the installation of wallabag...
